CHANGELOG.md

```diff
@@ -12,6 +12,9 @@
 - Android: handle device disconnect while the app is in the background
 - Improve send result view show relevant infos and options to make a new transaction or go back
 - Added an option in advanced settings to allow the app to start in testnet at the next restart.
+- Replaced radio button selection and "Next" button on buy / sell page with clickable cards / button
+- On screens less than or equal to 400px, exchange badges are hidden to save space
```

Comment:
2nd point is not clear that it is about buy&sell and it belongs to the point before.

How about changing to less specific "improve Buy&Sell view UI for small screen" or similar?